### PR TITLE
AC404 62 Code first approach

### PR DESCRIPTION
**Pull request by aantonic20**
Created and updated classes and models for code first database generation using lombok to remove boilerplate code.

### CODE REVIEW

File: paypro-android-backend/src/main/java/air/found/payproandroidbackend/core/models/CardBrand.java
Comment (timjuic) [inline paypro-android-backend/src/main/java/air/found/payproandroidbackend/core/models/CardBrand.java:1]: Good idea on using @Data insteaof @Getter and @Setter

File: paypro-android-backend/src/main/java/air/found/payproandroidbackend/core/models/Merchant.java
Comment (timjuic) [inline paypro-android-backend/src/main/java/air/found/payproandroidbackend/core/models/Merchant.java:1]: I think `ON UPDATE CASCADE ON DELETE NO ACTION` rule should be added on the property referencing the status table

**Review by timjuic — APPROVED** [review]
Well made code first entity model. I dont see any issues with it apart from referential rule CASCADE missing at one place

**Review by jivancic20 — APPROVED** [review]
Everything looks good, like Tim, I would also like to add the part ON UPDATE CASCADE ON DELETE NO ACTION in the Merchant class

**Pull request merged**